apps/explorer/src/index.css

```diff
@@ -14,7 +14,132 @@
     --steel-dark: theme('colors.steel.dark');
 }
 
-body {
-    @apply antialiased;
-    @apply font-sans text-offblack;
+@layer base {
+    body {
+        @apply antialiased;
+        @apply font-sans text-offblack;
+    }
+}
+
+@layer components {
+    .home-page-grid-container {
+        @apply grid grid-cols-1 gap-4;
+    }
+
+    @screen sm {
+        .home-page-grid-container {
+            @apply grid-cols-[200px,auto];
+            grid-template-areas:
+                'tps         tps'
+                'network     network'
+                'epoch       gas-price'
+                'checkpoint  gas-price'
+                'node-map    node-map'
+                'activity    activity'
```

Comment:
I feel pretty split about activity / validators / packages being part of this grid layout. Not really concerned about it for the scope of this PR, just feels like it might be best defined separately, but we can figure that out later too.

Reply:
Makes sense! I think there might be a case to keep this as part of the main grid layout with some stuff we're adding in the near future, but I haven't seen responsive designs so perhaps we can re-evaluate then :)